vllm/entrypoints/llm.py

```diff
@@ -349,9 +349,10 @@ def generate(
         outputs = self._run_engine(use_tqdm=use_tqdm)
         return LLMEngine.validate_outputs(outputs, RequestOutput)
 
+    @overload
     def chat(
         self,
-        messages: List[ChatCompletionMessageParam],
+        conversations: List[ChatCompletionMessageParam],
```

Comment:
We should note this as a (small) breaking change in the release for people who might be using kwargs for messages

Reply:
@aandyw I agree with Michael with this. Why did we need to rename this to conversations in the first place? I think messages fit better since a lot of people are used to the OpenAI format (`prompt` for completions, `messages` for chat completions.

Reply:
Makes sense. I'll change it back. I thought it would be more clear that it's possible to hold multiple separate conversations/messages